WordPress/gutenberg
# fix: Prevent duplicative offline status indicators

## What?

Prevent duplicative offline status indicators from displaying within nested
block lists for inner blocks.

## Why?

Fixes https://github.com/wordpress-mobile/gutenberg-mobile/issues/6468. The duplicate indicators are confusing.

## How?

Hoist the `OfflineStatus` indicator from the block list to the editor.
The block list is leveraged for inner blocks, which means it rendered
nested `OfflineStatus` indicators for blocks with inner blocks.

Additionally, the `editor` package feels like an appropriate location
for the offline detection component.

## Testing Instructions




1. Launch the editor.
1. Add a List block with items and a Gallery block with images.
1. Disable your device's network connection.
1. Verify only a single offline status indicator displays within the editor.

### Testing Instructions for Keyboard

N/A, now changes to navigation proposed.

## Screenshots or screencast 
N/A, no visual changes proposed.
